bsp/nuclei/gd32vf103_rvstar/board/board.c

```diff
@@ -32,6 +32,110 @@ extern void *_heap_end;
  */
 extern void _init(void);
 
+// Check pin assignment here https://doc.nucleisys.com/nuclei_board_labs/hw/hw.html
+
+void rt_hw_spi_drvinit(void)
+{
+    /* Comments below are sample configuration code for 
```

Comment:
Please remove the uneccesary code.  When you use macro condition to enable/disable code, then the code should be usefull or totally removed.

Reply:
Hi, this is necessary code, when I enable BSP_USING_SPI in menuconfig, then user need to configure the pinmux according to their pin assignment.

Reply:
As I commented in this function, I wish developer to adapt the pinmux by themselves according to their pin usage.

Reply:
If you like this code, please uncomment them. If you comment these codes, please remove it.

Reply:
Hi, @BernardXiong, what I leave here is just code comment for user to easily understand write what kind of code in this function. Maybe I can remove it and document it in README.md